private/src/styles/gutenberg.scss

```diff
@@ -140,22 +140,6 @@
   }
 }
 
-p.has-small-font-size {
```

Comment:
Similar to above, I'm hoping that these classes aren't required if they're provided by Gutenberg/theme.json variables.